### PR TITLE
TE-777: clean up fullscreen version of Modal

### PR DESCRIPTION
**Pull request by stephenkr**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-777)

### What **one** thing does this PR do?
Removes the height `100% - 1px` CSS to fix the fullscreen bug

### Any other notes
- Also had to remove the margin from the fullscreen modal to ensure it is 100% fullscreen.

__The `margin: 0 !important` fixes:__
screen shot 2018-08-08 at 12 56 06

__Preview of fullscreen modal with device toolbar open in Chrome devtools__
screen shot 2018-08-08 at 12 54 42

### CODE REVIEW

File: src/styles/semantic/themes/livingstone/modules/modal.variables [hunk 1]
```diff
@@ -27,7 +27,7 @@
 /* Responsive Widths */
 @fullScreenWidth: 100%;
 // Fixes the bug I describe at https://stackoverflow.com/questions/41453780/semantic-react-modal-weird-height-behavior/51426377#51426377
```
Comment (marcdavi-es) [inline src/styles/semantic/themes/livingstone/modules/modal.variables]: This comment can go.

Comment (stephenkr) [reply src/styles/semantic/themes/livingstone/modules/modal.variables]: True, i'll remove it.

File: src/styles/semantic/themes/livingstone/modules/modal.overrides [hunk 2]
```diff
@@ -48,6 +48,12 @@
   position: absolute;
   top: 0;
 
+  // This will remove the spacing around the modal when in fullscreen
+  // The selector had to be more specific for the rules to be applied.
```
Comment (marcdavi-es) [inline src/styles/semantic/themes/livingstone/modules/modal.overrides]: Dude let's lose these two comments. Just wanted to see the difference for the purposes of the PR :)

**Comment by marcdavi-es** [comment]
All good - let's just lose the comments.

File: src/styles/semantic/themes/livingstone/modules/modal.overrides (hunk 2 above)
Comment (stephenkr) [reply src/styles/semantic/themes/livingstone/modules/modal.overrides]: I'll remove them.

**Pull request merged**